### ISSUE TITLE
communication transparently data link

### ISSUE THREAD

**Issue by manotroll** [romanz/amodem#62]
It is possible to use it as a data link
Your project made something I was thinking about feasible

in this case it would be to open an audio link to send data and another to receive
or a phone call and use it as a modem and useful to get around places where the speed is low or where there is an additional charge for higher speeds

**Comment by ke4ahr** [comment]
Not really. A telephone modem has all the tricks for getting the most speed out of a 0-4 kHz speech window. It'd be easier to take the resistive hybrid out of a pair of telephone modems and then couple them to whatever media you plan to use to communicate as long as it isn't a phone line.

Otherwise, if it is a phone line, then just provide talk battery (-48 VDC @ 20 mA) and use a ring-down generator to give yourself a 33.6 kbit/s full-duplex line between two locations using V.92 modems.
